drivers/i2c/i2c_ite_enhance.c

```diff
@@ -408,6 +411,7 @@ static int enhanced_i2c_error(const struct device *dev)
 	} else if ((i2c_str & E_HOSTA_BDS_AND_ACK) == E_HOSTA_BDS) {
 		if (IT8XXX2_I2C_CTR(base) & E_ACK) {
 			data->err = E_HOSTA_ACK;
+			data->nack = 1;
```

Comment:
`nack` is a bool type.
```suggestion
			data->nack = true;
```

Reply:
Done.